libs/orders/src/lib/components/order-feedback/order-feedback.tsx

```diff
@@ -18,46 +18,7 @@ export const OrderFeedback = ({ transaction, order }: OrderFeedbackProps) => {
   const labelClass = 'font-bold text-black dark:text-white';
   if (!order) return null;
 
-  // Order on network but was rejected
-  if (order.status === OrderStatus.Rejected) {
-    return (
-      <p data-testid="error-reason">
-        {order.rejectionReason &&
-          t(`Reason: ${formatLabel(order.rejectionReason)}`)}
-      </p>
-    );
-  }
-
-  if (order.status === OrderStatus.Cancelled) {
-    return (
-      <div data-testid="order-confirmed">
-        <div className="grid grid-cols-1 md:grid-cols-2 gap-8 mb-8">
-          {order.market && (
-            <div>
-              <p className={labelClass}>{t(`Market`)}</p>
-              <p>{t(`${order.market.name}`)}</p>
-            </div>
-          )}
-        </div>
-        <div>
-          {transaction.txHash && (
-            <div>
-              <p className={labelClass}>{t('Transaction')}</p>
-              <a
```

Comment:
question (non blocking): Are we fine not showing the vega tx hash?

Reply:
we are showing the tx hash on line 72

Reply:
cancelled/rejected vs confirmed order were too similar - we would display the same component for both, the title would change based on the order status and if the order is rejected/stopped then we would display a reason